### ISSUE TITLE
R CMD check NOTE about Imports: RcppParallel

### ISSUE THREAD

**Issue by mlysy** [stan-dev/rstantools#99]
Upon `R CMD check`ing a package built with **rstantools** I get the following NOTE:

```r
 ❯ checking dependencies in R code ... NOTE
  Namespace in Imports field not imported from: ‘RcppParallel’
    All declared Imports should be used.
```

I see that the [**RcppParallel** setup documentation](https://rcppcore.github.io/RcppParallel/#r_packages) has an `importFrom` statement which should make this go away (it does for me locally, anyway).  There are also a few minuscule adjustments that could be made to the `Makevars[.win]` files to align more closely with what **RcppParallel** recommends.  

Are you OK with me experimenting a bit then making a PR?

**Comment by jgabry** [comment]
Sorry for the delay on this, I missed this issue back in the fall. If you still want to look into this and make a PR that would be great! 

(I'm also seeing this note when preparing to submit an rstantools update to CRAN so I might have to use the importFrom statement in rstantools' NAMESPACE too)

**Comment by mlysy** [comment]
OK sounds good!

**Comment by andrjohns** [comment]
For avoiding Importing `NOTE`s for both rstantools and RcppParallel in other packages I've been adding:
```
importFrom(rstantools,rstan_config)
importFrom(RcppParallel, RcppParallelLibs)
```

to the package's `NAMESPACE`

**Comment by mlysy** [comment]
That's very helpful, thanks

**Comment by andrjohns** [comment]
Closed by #115
